djangocms_navigation/admin.py

```diff
@@ -40,6 +51,13 @@
     LOCK_MESSAGE = _("You don't have permission to change this item")
 
 
+def proxy_model(obj):
+    versionable = versionables.for_content(PageContent)
```

Comment:
This doesn't look right, it's getting a proxy model for a page, I suspect that we don't need this at all as it is used when getting the edit link.

Reply:
The first thing to try would be to swap out the Pagecontent model for MenuContent, i'd want to work out why this is needed, it's replacing the object class but I'm not sure why. This is where comments come into their own, when we learn we will add comments.